x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -465,11 +466,9 @@ export const tasks: TelemetryTask[] = [
   {
     name: 'integrations',
     executor: async ({ transportRequest }) => {
-      const apmJobs = ['*-high_mean_response_time'];
-
       const response = (await transportRequest({
         method: 'get',
-        path: `/_ml/anomaly_detectors/${apmJobs.join(',')}`,
+        path: `/_ml/anomaly_detectors/${ML_GROUP_NAME_APM}-*`,
```

Comment:
I don't think you need the wildcard since this is returning all jobs with the given group
```suggestion
        path: `/_ml/anomaly_detectors/${ML_GROUP_NAME_APM}`,
```

Reply:
Is that right? Before we were `*-high_mean_response_time`. Also, we should probably keep that previous value as well since we'll want to count both the old and new jobs.